yoticlient.go

```diff
@@ -88,10 +88,15 @@ func (client *Client) GetUserProfile(token string) (userProfile UserProfile, fir
 }
 
 func (client *Client) getAPIURL() string {
-	if client.apiURL != "" {
-		return client.apiURL
+	if client.apiURL == "" {
+		if os.Getenv("YOTI_API_URL") != "" {
+			return os.Getenv("YOTI_API_URL")
+		} else {
+			return apiDefaultURL
+		}
```

Comment:
```suggestion
		if value, ok := os.LookupEnv("YOTI_API_URL"); ok {
			return value
		}
		return apiDefaultURL
```

Reply:
Have updated in 31a425c, thought it was slightly clearer using `exists` instead of `ok`, and did a little restructuring to make the logic clearer, and also applied to the other place where we read this variable